src/SDKs/Automation/Management.Automation/Microsoft.Azure.Management.Automation.csproj

```diff
@@ -11,6 +11,7 @@
     <AssemblyName>Microsoft.Azure.Management.Automation</AssemblyName>
     <PackageTags>Automation;Runbook;</PackageTags>
     <PackageReleaseNotes>Major version release: SDK rebuilt using the Swagger specs - Changes in the namespaces and interfaces. Still in preview mode and contains breaking changes.</PackageReleaseNotes>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
@D1v38om83r Can you bump up the version to  <Version>3.0.1-preview</Version>

Reply:
Added a new commit for the same.

Reply:
@D1v38om83r if you are planning to release then bump up the version.
I only see changes related to tests plus I am not sure if you want to release a new version with only test related changes.
Whenever you bump up the version, you need to also add new release notes.
Please remove line 14. If you are still getting duplicate attribute errors you need to do the following:

Can you do the following in the below order:
1)	Pull latest from upstream (if you have already done this good)
2)	Execute msbuild.exe build.proj from the repo root
a.	This will update the build tools in the current repo
3)	Generate your SDK
4)	Build you sdk at least one from command line of IDE
a.	Msbuild.exe build.proj /t:build /p:Scope=SDKs\Automation
5)	Update the PR